parse_event_logs.py

```diff
@@ -180,6 +180,33 @@ def output_utilizations(self, prefix):
     self.__write_utilization_summary_file(
       process_system_cpu_utilizations, "%s_%s" % (prefix, "cpu_process_system_utilization"))
 
+  def output_stage_resource_metrics(self, filename):
+    """ Writes a file with resource metrics at the granularity of stages. """
```

Comment:
Can you make this more descriptive -- something like "Writes a single file with the CPU, network, and disk resources used by each stage"?